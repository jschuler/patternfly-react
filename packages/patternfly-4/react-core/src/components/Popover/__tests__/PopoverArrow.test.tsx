// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { PopoverArrow } from '../PopoverArrow';

describe('PopoverArrow test', () => {
  it('PopoverArrow should match snapshot', () => {
    const view = mount(
      <PopoverArrow
        className={"''"}
      />);
    expect(view).toMatchSnapshot();
  });
});
