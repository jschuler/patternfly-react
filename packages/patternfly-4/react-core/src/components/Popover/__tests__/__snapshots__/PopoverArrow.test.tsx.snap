// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopoverArrow test PopoverArrow should match snapshot 1`] = `
<Component
  className="''"
>
  <div
    className="pf-c-popover__arrow ''"
  />
</Component>
`;
